docs/community/best_paper_award.md

```diff
@@ -27,6 +27,9 @@ The best deal.II-based paper from 2023 is the following:
 > [[DOI: 10.1016/j.cma.2023.115983]](http://doi.org/10.1016/j.cma.2023.115983)
 > [[URL]](https://www.sciencedirect.com/science/article/pii/S0045782523001068)
 > [[Preprint]](https://arxiv.org/abs/2207.12460)
+>
+> ![An image from the publication](../large-assets/best-paper-award/2023/figure_res_baseline_active_tension.png)
```

Comment:
OK, this link does not work. What is the correct link to reference these pictures? Does it have to be a global URL, or should I reference a relative path? In either case, which one is it?

Reply:
@bangerth Because it is `large_assets`. I have fixed the typo!

Reply:
Oh, good grief :-( I played with it for fifteen minutes last night and just couldn't figure it out. I totally missed that issue...